daml_dit_ddit/common.py

```diff
@@ -41,7 +41,10 @@ def accept_dabl_meta(data: bytes) -> 'PackageMetadata':
 def load_dabl_meta() -> 'PackageMetadata':
     try:
         with open(DABL_META_NAME, "r") as f:
-            return accept_dabl_meta(f.read().encode())
+            dabl_meta = accept_dabl_meta(f.read().encode())
+            check_experimental(dabl_meta)
```

Comment:
I inserted this here because `load_dabl_meta()` is eventually called by (almost) every other ddit subcommand, so I did it in one place here to avoid having to do it separately in each other instance. 

I'm still not totally sure this is the right spot in the hierarchy for this check to go, though, so let me know if you have any other suggestions.

Reply:
I think that's good.